templates/search/search_clients_daily_v8/query.sql

```diff
@@ -29,6 +29,19 @@ CREATE TEMP FUNCTION get_search_addon_version(active_addons ANY type) AS (
   )
 );
 
+-- Take array of experiment structs and return union of experiments.key_value arrays
+-- This is a workaround for the inability to use ARRAY_CONCAT_AGG as an analytical function
+CREATE TEMP FUNCTION union_experiments(list ANY TYPE) AS (
+  (
+    SELECT udf_dedupe_array(ARRAY_CONCAT_AGG(key_value))
+    FROM (
+      SELECT 1 AS val, key_value
+      FROM UNNEST(list)
+    )
+    GROUP BY val
+  )
+);
```

Comment:
this can be more efficiently written as:
```sql
CREATE TEMP FUNCTION union_experiments(list ANY TYPE) AS (
  (
    SELECT
      udf_dedupe_array(ARRAY_CONCAT_AGG(key_value))
    FROM
      UNNEST(list)
  )
);
```

Reply:
there is an implicit `GROUP BY TRUE` here